pkg/services/ngalert/api/authorization.go

```diff
@@ -304,6 +315,7 @@ func (api *API) authorize(method, path string) web.Handler {
 					ac.EvalPermission(ac.ActionAlertingRuleUpdate, scope),
 					ac.EvalPermission(ac.ActionAlertingRuleCreate, scope),
 					ac.EvalPermission(ac.ActionAlertingRuleDelete, scope),
+					ac.EvalPermission(ac.ActionAlertingProvisioningSetStatus),
```

Comment:
This should be outside of the `ac.EvalAny`, no?

Reply:
yes!